config/database.yml

```diff
@@ -0,0 +1,31 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```

Comment:
- [ ] Great job working on the working on setup but I think it would be great if you set up your project with PostgreSQL since I could see sqlite3 in your database so kindly I recommend setting up with PostgreSQL instead of sqlite3 since after trying to create the DB I faced this error 💯🚀


![image](https://github.com/Tayyab-Kamboh/blog-app/assets/58553711/57056f03-41d7-4b3e-9463-796cea45fc8d)

Reply:
@Reem-lab Is it necessary to use postgre? because I have been facing this error all day long while trying to migrate files even when the SQL tool says that the connection has been established: 
```
PS C:\Users\tayya\blog-app> rails db:migrate        
rails aborted!
ActiveRecord::ConnectionNotEstablished: connection to server at "localhost" (::1), port 5432 failed: server closed the connection unexpectedly
        This probably means the server terminated abnormally
        before or while processing the request.


Caused by:
PG::ConnectionBad: connection to server at "localhost" (::1), port 5432 failed: server closed the connection unexpectedly
        This probably means the server terminated abnormally
        before or while processing the request.
```